tests/test_io/test_datasets.py

```diff
@@ -149,3 +150,25 @@ def test_statsbomb_get_pass_custom_teamsheets() -> None:
     assert teamsheets["Home"].teamsheet.at[0, "pID"] == 999999
     assert "custom_col" in teamsheets["Home"].teamsheet.columns
     assert "my_col" in teamsheets["Away"].teamsheet.columns
+
+
+@pytest.mark.unit
+def test_dfl_get() -> None:
+
+    dataset = DFLDataset()
+    event_objects, position_objects = dataset.get()
+    assert isinstance(event_objects[0]["firstHalf"]["Home"], Events)
+    assert isinstance(event_objects[0]["secondHalf"]["Away"], Events)
+    assert isinstance(event_objects[1]["Home"], Teamsheet)
+    assert isinstance(event_objects[1]["Away"], Teamsheet)
+    assert isinstance(event_objects[2], Pitch)
+
+    assert isinstance(position_objects[0]["firstHalf"]["Home"], XY)
+    assert isinstance(position_objects[0]["secondHalf"]["Away"], XY)
+    assert isinstance(position_objects[1]["secondHalf"], Code)
+    assert isinstance(position_objects[1]["firstHalf"], Code)
+    assert isinstance(position_objects[2]["secondHalf"], Code)
+    assert isinstance(position_objects[2]["firstHalf"], Code)
+    assert isinstance(position_objects[3]["Home"], Teamsheet)
+    assert isinstance(position_objects[3]["Away"], Teamsheet)
+    assert isinstance(position_objects[4], Pitch)
```

Comment:
appreciate the effort, but how long does this take? feels like it may blow up our unit test suite from 8 seconds capable of running offline to 1 minute requiring internet connection

Reply:
I agree on the duration part. This test alone takes about 1:15 min on my machine. I will remove the test for now. I think for this dataset, a good test could be checking the respective links for valid responses as internet connection is already required e.g., for the `StatsBombOpenDataset` tests. 
However, I expect the final link to change (probably remove the private token) once the paper is published. So this could be added in a future fix?

Reply:
yes, let's do it that way! probably make sense to split up our test-suite at some point with fast unit tests and a dedicated test battery for the parsing that might take longer, so they only need to be tested when working on that module or on pull requests. feel free to open another PR or issue on that